plugins/ngrok/auth_credentials.go

```diff
@@ -32,21 +32,8 @@ func Credentials() schema.CredentialType {
 					},
 				},
 			},
-			{
-				Name:                fieldname.APIKey,
-				MarkdownDescription: "API Key used to authenticate to ngrok API.",
-				Optional:            true,
-				Secret:              true,
-				Composition: &schema.ValueComposition{
-					Length: 48,
-					Charset: schema.Charset{
-						Uppercase: true,
-						Lowercase: true,
-						Digits:    true,
-					},
-				},
-			},
 		},
+		DefaultCredential:  true,
 		DefaultProvisioner: provision.TempFile(ngrokConfig, provision.Filename("config.yml"), provision.AddArgs("--config", "{{ .Path }}")),
```

Comment:
same as above, we could use the AUTH_TOKEN env var: https://ngrok.com/docs/ngrok-agent/#supported-env-variables

Reply:
See issue: https://github.com/1Password/shell-plugins/issues/216